blocks/README.md

```diff
@@ -125,6 +127,7 @@ Registers a new block provided a unique slug and an object defining its behavior
 
 - `title: string` - A human-readable [localized](https://codex.wordpress.org/I18n_for_WordPress_Developers#Handling_JavaScript_files) label for the block. Shown in the block picker.
 - `icon: string | WPElement` - Slug of the [Dashicon](https://developer.wordpress.org/resource/dashicons/#awards) to be shown in the control's button, or an element if you choose to render your own SVG.
+- `category: string` - Id of the block's category. The category is used to organize the blocks in the block inserter.
```

Comment:
I think we should validate the `settings` object and check the mandatory attributes (title, icon, category), but since it's not done yet for any setting. I'd leave this to its own PR.